src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -35,12 +36,52 @@ public Settings(IConfigurationSource source)
             JaegerExporterAgentHost = source.GetString(ConfigurationKeys.JaegerExporterAgentHost) ?? "localhost";
             JaegerExporterAgentPort = source.GetInt32(ConfigurationKeys.JaegerExporterAgentPort) ?? 6831;
 
+            LoadTracerAtStartup = source.GetBool(ConfigurationKeys.LoadTracerAtStartup) ?? true;
+
+            ConsoleExporterEnabled = source.GetBool(ConfigurationKeys.ConsoleExporterEnabled) ?? true;
+
+            EnabledInstrumentations = Enum.GetValues(typeof(InstrumentationType)).Cast<InstrumentationType>().ToList();
+            var disabledConfigurations = source.GetString(ConfigurationKeys.DisabledInstrumentations);
+            if (disabledConfigurations != null)
+            {
+                foreach (var instrumentation in disabledConfigurations.Split(separator: ','))
+                {
+                    if (Enum.TryParse(instrumentation, out InstrumentationType parsedType))
+                    {
+                        EnabledInstrumentations.Remove(parsedType);
+                    }
+                    else
+                    {
+                        // TODO replace with proper logging
+                        Console.WriteLine($"Could not parse instrumentation \"{instrumentation}\". Skipping...");
```

Comment:
You can use `ConsoleLogger` with `ILogger` interface. Later it's easier to replace implementing type.

Reply:
Done